blog-csoar/2024-11-20-content.md

```diff
@@ -0,0 +1,47 @@
+
+### Integrations
+
+* [New] [Sumo Logic Automation Tools](/docs/platform-services/automation-service/app-central/integrations/sumo-logic-automation-tools)
+* [New] [Google Chat](/docs/platform-services/automation-service/app-central/integrations/google-chat)
+* [New] [Malwarebytes Oneview](/docs/platform-services/automation-service/app-central/integrations/malwarebytes-oneview)
+* [New] [Silent Push](/docs/platform-services/automation-service/app-central/integrations/silent-push)
+* [New] [VirusTotal V3](/docs/platform-services/automation-service/app-central/integrations/virustotal-v3)
+* [Updated] [Azure AD](/docs/platform-services/automation-service/app-central/integrations/azure-ad)
+* [Updated] [Microsoft EWS](/docs/platform-services/automation-service/app-central/integrations/microsoft-ews)
+* [Updated] [Netskope V2](/docs/platform-services/automation-service/app-central/integrations/netskope-v2)
+* [Updated] [Cloudflare](/docs/platform-services/automation-service/app-central/integrations/cloudflare)
+* [Updated] [ConnectWise Manage](/docs/platform-services/automation-service/app-central/integrations/connectwise-manage)
+* [Updated] [Microsoft Sentinel](/docs/platform-services/automation-service/app-central/integrations/microsoft-sentinel)
+* [Updated] [CrowdStrike Falcon](/docs/platform-services/automation-service/app-central/integrations/crowdstrike-falcon)
+* [Updated] [Cortex XDR](/docs/platform-services/automation-service/app-central/integrations/cortex-xdr)
+* [Updated] [Slack](/docs/platform-services/automation-service/app-central/integrations/slack)
+* [Updated] [Atlassian Opsgenie](/docs/platform-services/automation-service/app-central/integrations/atlassian-opsgenie)
+* [Updated] [Freshservice](/docs/platform-services/automation-service/app-central/integrations/freshservice)
+* [Updated] [Atlassian Jira V2](/docs/platform-services/automation-service/app-central/integrations/atlassian-jira-v2)
+* [Updated] [Sumo Logic Notifications by Gmail](/docs/platform-services/automation-service/app-central/integrations/sumo-logic-notifications-by-gmail)
+* [Updated] [AWS EKS](/docs/platform-services/automation-service/app-central/integrations/aws-eks)
+* [Updated] [AWS EC2](/docs/platform-services/automation-service/app-central/integrations/aws-ec2)
+* [Updated] [VirusTotal](/docs/platform-services/automation-service/app-central/integrations/virustotal)
+* [Updated] [Sumo Logic Cloud SIEM](/docs/platform-services/automation-service/app-central/integrations/sumo-logic-cloud-siem)
+* [Updated] [Microsoft OneDrive](/docs/platform-services/automation-service/app-central/integrations/microsoft-onedrive)
+* [Updated] [APIVoid](/docs/platform-services/automation-service/app-central/integrations/apivoid)
+* [Updated] [Gmail](/docs/platform-services/automation-service/app-central/integrations/gmail)
+* [Updated] [URLScan.io](/docs/platform-services/automation-service/app-central/integrations/urlscan-io)
```

Comment:
Hello @rishav-sumo-dev , thanks for submitting this PR! It looks like the build broke because it doesn't recognize `/docs/platform-services/automation-service/app-central/integrations/urlscan-io` as a valid link.